tahoe_sites/urls.py

```diff
@@ -6,6 +6,6 @@
 from tahoe_sites.views import OrganizationViewSet
 
 router = routers.SimpleRouter()
-router.register(r'tahoe_sites_organizations', OrganizationViewSet, basename='tahoe_sites_organization')
+router.register(r'tahoe-sites/api/v1/organizations', OrganizationViewSet, basename='tahoe_sites_organization')
```

Comment:
This is a completely cosmetic change.

```suggestion
router.register('api/v1/organizations', OrganizationViewSet, basename='tahoe_sites_organization')
```

Please consider adding the `tahoe-sites` prefix in the [`apps.py` file](https://github.com/appsembler/tahoe-sites/blob/db3ff04994612e919df8e189699a5e1e01ea0b2a/tahoe_sites/apps.py#L15-L23):




```py
        'url_config': {
            'lms.djangoapp': {
                'namespace': 'tahoe_sites',
                'app_name': 'tahoe_sites',
                'regex': '^tahoe-sites/', # added by omar
            },
            'cms.djangoapp': {
                'namespace': 'tahoe_sites',
                'app_name': 'tahoe_sites',
                'regex': '^tahoe-sites/', # added by omar
            }
        },
```

Reply:
I always like your cosmetic changes @OmarIthawi , thank you!
Done